test/promql/tests/recording/workspaces_availability_recording_rules_test.yaml

```diff
@@ -0,0 +1,25 @@
+evaluation_interval: 1m
+
+rule_files:
+  - workspaces_service_availability_recording_rules.yaml
+
+tests:
+  - interval: 1m
+    name: RSExporterTest
```

Comment:
What's `RSExporter`?

Reply:
I can rename but was not sure if that made a diff since most test have the same name, if this can be unique to Workspaces I rather rename

Reply:
Renamed